### DISCUSSION TITLE
Problem in GitHub Student Devloper pack

### DISCUSSION THREAD

**Discussion by lakshit-html** [community/community#143775] [GitHub Education]
I am facing a problem to get email from git hub community, I had recently applied for github student devloper pack and I got approval but I am waiting from 2 days but I am not getting any email to access these offers 
   **Can anyone suggest me how to access those offers**

I am a new frontend  devloper  recently joined GitHub

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @lakshit-html , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Comment by davevad93** [comment]
Hi @lakshit-html ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @lakshit-html was approved and your benefits are now available.`

Don't worry, the waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
